keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -343,35 +343,86 @@ are properly reloading tokens by:
 1.  Add annotation to audit events for legacy and stale tokens including
     necessary information to locate problematic client.
 
+### Test Plan
+
+#### TokenRequest/TokenRequestProjection
+
+- Unit tests
+- E2E tests
+  - Projected jwt tokens are correctly mounted. (conformance test)
+  - The owner and mode of projected tokens are correctly set
+  - In-cluster clients work with Token rotation
+
+#### RootCAConfigMap
+
+#### BoundServiceAccountTokenVolume
+
+- Unit tests
+-  An upgrade test
+  1. Create pod A with feature disabled where pod A is working and a secret volume
+    is mounted
+  2. Enable feature where pod A continue working
+  3. Create pod B and it is working and projected volumes are mounted
+
 ### Graduation Criteria
 
-#### Alpha->Beta
+#### TokenRequest/TokenRequestProjection
+
+| Alpha | Beta | GA |
+| --- | --- | --- |
+| 1.10 | 1.12 | 1.20 |
+
+##### Alpha->Beta
+
+##### Beta->GA
+
+- [x] In use by multiple distributions
+- [x] Approved by PRR and scalability
+- [x] Known bugs fixed
+- [x] Tests passing
+  - [x] E2E test [ServiceAccounts should mount projected service account token when requested](https://k8s-testgrid.appspot.com/sig-auth-gce#gce)
+  - [x] E2E test [ServiceAccounts should set ownership and permission when RunAsUser or FsGroup is present](https://k8s-testgrid.appspot.com/sig-auth-gce#gce)
+  - [x] E2E test [ServiceAccounts should support InClusterConfig with token rotation](https://k8s-testgrid.appspot.com/sig-auth-gce#gce-slow)
+
+#### RootCAConfigMap
```

Comment:
update "Will enabling / using this feature result in increasing size or count of the existing API objects?" PRR section, controller creates one additional configmap per namespace

Reply:
done